plc4j/integrations/apache-kafka/pom.xml

```diff
@@ -33,8 +33,17 @@
 
   <properties>
     <kafka.version>2.5.0</kafka.version>
+    <kafka.connect.maven.plugin.version>0.11.3</kafka.connect.maven.plugin.version>
   </properties>
 
+  <repositories>
+    <repository>
```

Comment:
As this is used to download the plugin, we need to also specify a "pluginRepository" (Just a duplicate with different element name) ... otherwise the build will fail for people who haven't downloaded the plugin before.